include/jemalloc/internal/arena_inlines_b.h

```diff
@@ -298,6 +298,61 @@ arena_dalloc_large(tsdn_t *tsdn, void *ptr, tcache_t *tcache, szind_t szind,
 	}
 }
 
+/*
+ * Internal.
+ *
+ * Find the region index of a pointer.
+*/
```

Comment:
nit: format (the stars should align). Please check the comments format of other places too. Also no need to say "Internal" in this case, since the `_impl` name implies that too.

Reply:
modified.